prjxray/util.py

```diff
@@ -73,7 +80,7 @@ def slice_xy():
 def get_roi():
     from .db import Database
     (x1, x2), (y1, y2) = roi_xy()
-    db = Database(get_db_root())
+    db = Database(get_db_root(), get_part())
```

Comment:
Since get_part() is basically only getting an environment variable why can't this be done in the Database class. This would save many calls of this function for example in the `top.py` scripts in many fuzzers

Reply:
The fact is that the Database is accessed also outside of prjxray, where no env variable is set (e.g. in arch-defs). Therefore, there is the need to get it as input to create the Database object.

Reply:
Of course I could avoid adding that to the prjxray function calls (as the Database init function could look for the env variable, and if it is not present, and the argument is also `None`, than it could throw an error).

Reply:
I know @litghost is not a big fan of environment variables so maybe it's better to leave it the way you have it currently. I just see that like with any API changes, the number of fuzzers that require adjustment is quite big. Other than that I don't see any problem with having to call `get_part()`.

Reply:
@acomodi made the right change, and the amount of churn within prjxray isn't that concerning.